tests/conftest.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (C) 2017-2022  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+from typing import Union
+
+import pytest
+import numpy as np
+import sklearn
+from sklearn.datasets import make_multilabel_classification
+from sklearn.model_selection import train_test_split
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.linear_model import LogisticRegression
+
+from cleanlab import count
+from cleanlab.benchmarking.noise_generation import (
+    generate_noise_matrix_from_trace,
+    generate_noisy_labels,
+)
+from cleanlab.internal.util import train_val_split
+
+
+def multilabel_py(y):
+    unique_labels = np.unique(y, axis=0)
+    n = y.shape[0]
+    multilabel_counts = [
+        np.sum(np.prod([label[i] == y[:, i] for i in range(y.shape[1])], axis=0))
+        for label in unique_labels
+    ]
+    return np.array(multilabel_counts) / float(n)
+
+
+DATASET_KWARGS = {
+    "n_samples": 500,
```

Comment:
seems overly large? We need unit tests to run really fast. 

Code used for performance estimation or benchmarking can go in examples/ repo instead. For now can just be a comment in the PR description too if it's useful to facilitate reviewing.

Reply:
Yup, it's too large for unit tests, but better for benchmarking.
The test runtime isn't affected too drastically right now, but the tests should pass with far fewer samples.

Reply:
Let's just plan to have your benchmark code go into cleanlab/examples/ eventually. So here let's focus on minimizing runtime of the unit tests. It's a bit cumbersome to revisit individual unit tests in the future to speed them up, so we should always aim to ensure each test runs fast when we first introduce them.

Reply:
@elisno What is the plan with this file?

Reply:
I think we should only have tests in: tests/test_multilabel_classification.py

Anything that involves sample-size 500 or does not fit well in that file is probably not a unit test and instead an overall performance benchmark. Those we can put in the examples/ repo as I mentioned above.